doc/Manual.md

```diff
@@ -357,6 +357,29 @@ You can then easily initialize a repository that uses your Amazon S3 as a backen
 For an S3-compatible repository without TLS available, use the alternative URI
 protocol `s3:http://server:port/bucket_name`.
 
+# Create a Minio Server repository
```

Comment:
I think this would fit better as a sub-heading to the s3 backend example, could you add another `#`?